README.md

```diff
@@ -32,20 +32,24 @@ which is required by the automation voice.
 
        npm install -g @bocoup/windows-sapi-tts-engine-for-automation
```

Comment:
I was unable to run this with the published package.

I had to do
```
chmod +x ./bin/at-driver
```
and `./bin/at-driver` in place of all instances of `at-driver`

Reply:
Since this problem exists on the `main` branch, I've submitted a fix via [a separate pull request](https://github.com/w3c/aria-at-automation-driver/pull/48).